src/Instrumentation/PDO/src/PDOInstrumentation.php

```diff
@@ -73,22 +76,41 @@ public static function register(): void
                     }
 
                     self::end($exception);
+                    
+                    $attributes = $pdoTracker->get($result);
+                    $parent = Context::getCurrent();
+
+                    $instrumentation->meter()
+                        ->createUpDownCounter('db.client.connection.count', '1')
```

Comment:
Should we be creating a new instrument each time, or one that is reused?

Reply:
All examples I found always created new meter. I assume this is a cheap operation. Re-using that would be much more complicated I think.